test/Microsoft.AspNetCore.Hosting.Tests/WebHostBuilderTests.cs

```diff
@@ -307,6 +307,7 @@ public void UseBasePathConfiguresBasePath()
         [Fact]
         public void RelativeApplicationBaseAreResolved()
         {
+            // Arrange & Act
```

Comment:
none of the tests is using this kind of comments (I personally don't see the value they add)